.github/workflows/windows-ci.yml

```diff
@@ -6,12 +6,12 @@ on:
       os-version:
         description: 'OS version to run the workflow'
         required: false
-        default: 'windows-13'
+        default: 'windows-2019'
         type: string
       vs-toolset:
         description: 'Visual Studio toolset to use'
         required: false
-        default: 'v142'
+        default: ''
```

Comment:
The defaults on Fast DDS are different: https://github.com/eProsima/Fast-DDS/blob/master/.github/workflows/reusable-windows-ci.yml#L37-L43